composer.json

```diff
@@ -14,22 +14,31 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php-http/discovery": "^0.2.0",
+        "php": ">=5.5",
+        "php-http/discovery": "^0.6.1",
```

Comment:
I think our discover layer can just be replaced by puli discovery. The symfony bundle provides a service for it. See how it is used in our discovery.